server/src/domain/system-config/system-config.core.ts

```diff
@@ -341,19 +342,19 @@ export class SystemConfigCore {
     if (force || !this.configCache) {
       try {
         const file = await this.repository.readFile(filepath);
-        const json = JSON.parse(file.toString());
+        const config = loadYaml(file.toString()) as any;
```

Comment:
js-yaml has this option:

> `json` (default: false) - compatibility with JSON.parse behaviour. If true, then duplicate keys in a mapping will override values rather than throwing an error.

I don't think we need it, right?

Reply:
Oh I didn't realize you could unit test it so easily, but that works too.